### DISCUSSION TITLE
Standard adókódok tartalmi módosulása

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#35] [Kérdések és válaszok - Gépi interfész] [answered]
Kedves NAV Szakértők,

Időben változhat-e tartalmában a standard adókód?

Példa (teljesen hipotetikus, csak a változás lényegére próbálunk rámutatni)

**1. Kiindulási állapot:** A1 standard adókód árubeszerzés 27%

**2. Új standard adókódok kerülnek közzétéve, melyben az A1 tartalma megváltozik, ami kétféle módon történhet:**

Opció 1: A1: standard adókód mezőgazdasági termékbeszerzés 27%, A2: standard adókód nem-mezőgazdasági termékbeszerzés 27% VAGY

Opció 2: A1: marad, ami volt: A1 standard adókód árubeszerzés 27% és 2 új adókód: A2: mezőgazdasági termékbeszerzés 27%, A3: nem-mezőgazdasági termékbeszerzés 27%

Milyen módon lesz kezelve a standard adókódok időbeli tartalmi változása: OPció 1 vagy Opció 2 szerint?

Előre is köszönjük a választ!

**Comment by renced42** [comment]
Kedves @szalaierika 

Az adókódnak egyedi jelentése van, ha változik az mást fog jelenteni. 
Általában ugyanabba a sorba nem fog más tartalom kerülni, ha mégis akkor az legfeljebb évente 1x változik, tehát év közben nem. A változások év végén publikálódnak ha lesznek.

**Reply by szalaierika** [reply]
Kedves Renced42 !

Köszi szépen a válaszod.

Jól értem, hogy elképzelhetó olyan eset, hogy ugyanannak a standard adókódnak egyik évről a másikra megváltozik a jelentése?

Ez azért fontos, mert olyan rendszerek esetében, ahol az adókód nem időfüggően állítható be ez esetleg problémás lehet.

Köszönöm előre.

**Comment by renced42** [comment, marked as answer]
Kedves @szalaierika 

Nem, ugyanaz az adókód ugyanazt fogja jelenteni. Ha változik a jelentése változik az adókód is.

**Reply by szalaierika** [reply]
Nagyszerű, köszi szépen a megerősítést!
